### PR TITLE
Move redirectToLogin to dhis2 utils module

### LINKED ISSUE #58: Refactor loginurl redirection
From the [PR](https://github.com/EyeSeeTea/dataset-configuration/pull/56)

we have this redirection to loginUrl here and in redirectToLogin method. The url path and the code that makes the redirection is redundant. Can we maybe extract this redirection method into an utils function? Then in the sessiondialog, redirectToLogin doesn't need to be a method hardcoded but a property that we pass to the component SessionDialog. What do u think?

### PR DESCRIPTION
**Pull request by tokland**
Closes #58

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -80,9 +82,10 @@ function startApp(d2) {
 // can use it to access the api, translations etc.
 getManifest('./manifest.webapp')
     .then(manifest => {
-        const baseUrl = process.env.NODE_ENV === 'production' ? manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        const baseUrl = process.env.NODE_ENV === 'production' ?
+            manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        config.siteUrl = baseUrl;
```
Comment (adrianq) [inline src/app.js]: siteUrl instead of baseUrl?

Comment (tokland) [inline src/app.js]: I was also perplexed.  It turns out `config.siteUrl` is used interally by d2 and must be set to `base/api/VERSION`. So I needed to create another key to point to the main site, thus the name.

Comment (adrianq) [inline src/app.js]: baseUrl is used internally you meant?

Comment (tokland) [inline src/app.js]: Internally, right. Note that this `config` comes from `import { init, config, getUserSettings, getManifest } from 'd2/lib/d2';`, it's a internal structure of d2.

Comment (adrianq) [inline src/app.js]: Ok, capito. You wrote siteUrl and this confused me. Merging...

Comment (tokland) [inline src/app.js]: Sorry, right, `baseUrl` is d2's, `siteUrl` is our new key.

**Pull request merged**